tests/Properties/ByteString.hs

```diff
@@ -4,7 +4,16 @@
 -- License     : BSD-style
 
 {-# LANGUAGE CPP #-}
+{-# LANGUAGE AllowAmbiguousTypes #-}
 {-# LANGUAGE ViewPatterns #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- We need @AllowAmbiguousTypes! in order to be able to use @TypeApplications@
+-- to disambiguate the desiered instance of class methods whose instance cannot
+-- be inferred from the caller's context.  We would otherwise have to use
+-- proxy arguments.  Here the 'RdInt` class methods used to generate tests for
+-- all the various 'readInt' types require explicit type applications.
```

Comment:
```suggestion
-- We need @AllowAmbiguousTypes@ in order to be able to use @TypeApplications@
-- to disambiguate the desired instance of class methods whose instance cannot
-- be inferred from the caller's context.  We would otherwise have to use
-- proxy arguments.  Here the 'RdInt' class methods used to generate tests for
-- all the various 'readInt' types require explicit type applications.
```